### ISSUE TITLE
SDDM fonts (text) not visible

### ISSUE THREAD

**Issue by nekromoff** [sddm/sddm#892]
Fonts or texts are not visible at all. When I type password, this will be similarly invisible.

No texts visible for session type (plasma, Gnome ...) or keyboard description:
![obrazok](https://user-images.githubusercontent.com/8550349/30457367-6e164f66-99a7-11e7-9192-37e6e698b7dd.png)

Selected password, but no visible characters (but they are written correctly as I can log in):
![obrazok](https://user-images.githubusercontent.com/8550349/30457396-86d7aa54-99a7-11e7-9ff7-704165af24d3.png)

**Comment by nekromoff** [comment]
When testing with greeter, same thing is happening. This happens for all themes (breeze, maui, default)

**Comment by nekromoff** [comment]
Also see:
https://stackoverflow.com/questions/45307518/sddm-fonts-not-loading/46228578#46228578

**Comment by anatolyborodin** [comment]
Interesting though, if you click the place with the mouse, where the "show password" eye icon should be, the password text is visible (or the grey "Password" is shown). But if you click it again, the text disappears.

The eye icon is invisible during the log in, but visible on the lock screen.

The text cursor is always visible and moving.

PS The letters on the visual keyboard are invisible too.

**Comment by nekromoff** [comment]
Yes, once I click the "eye", then the password is displayed correctly in
plain text.

**Comment by diamond2016** [comment]
I have similar issue. Sddm does not show labels username even the hostname (welcome to...). I have tried to reinstall fonts but no change.

**Comment by anatolyborodin** [comment]
I'm afraid, the problem can be in some related QT or KDE library: I've tried to downgrade sddm and sddm-theme-breeze to the versions from zesty (0.13.0-1ubuntu5 and 4:5.9.4-0ubuntu1 correspondently), but the problem is still there.

KDE version 4:5.10.5-0ubuntu1, QT version 5.9.1+dfsg-5ubuntu2.

PS The user icons are also invisible.

**Comment by diamond2016** [comment]
KDE Plasma 5.10.5 KF5 5.37.0 QT 5.9.1 kernel 4.12.12 (linux from scratch)

**Comment by pierre-labastie** [comment]
Does any of you have an English locale? For me, my locale is fr_FR.UTF-8. If I run
`sddm-greeter --test-mode`
I have the same issue as described above (everything works except there is no text). If I run
`LANG=C sddm-greeter --test-mode` or
`LANG=en_US.UTF-8 sddm-greeter --test-mode` or
`LANG=en_GB.UTF-8 sddm-greeter --test-mode`
the text is here (in English of course). So there is a problem with translations...
Two things that may be of importance:
- I have Qt in /opt/qt5, while sddm is in /usr (could explain that translation files are not found)
- I have tried to change TranslationDirectory to absolute (/usr/share/sddm/translations) in the metadata.desktop of elarun and to run that theme. No success.

**Comment by nekromoff** [comment]
Yes! Mine is:
```
LANG=sk_SK.UTF-8
LANGUAGE=en_US
LC_CTYPE="sk_SK.UTF-8"
LC_NUMERIC=sk_SK.UTF-8
LC_TIME=sk_SK.UTF-8
LC_COLLATE="sk_SK.UTF-8"
LC_MONETARY=sk_SK.UTF-8
LC_MESSAGES="sk_SK.UTF-8"
LC_PAPER=sk_SK.UTF-8
LC_NAME=sk_SK.UTF-8
LC_ADDRESS=sk_SK.UTF-8
LC_TELEPHONE=sk_SK.UTF-8
LC_MEASUREMENT=sk_SK.UTF-8
LC_IDENTIFICATION=sk_SK.UTF-8
LC_ALL=
```

Your solution with switching to English works, thanks!

**Comment by nekromoff** [comment]
And suddenly lock screen is showing everything (same issue as with SDDM) in order as well! Great.

**Comment by diamond2016** [comment]
I have IT_it.UTF-8 and the same configuration of pierre-labastie. I've tried to set LANG and /etc/locale.conf to en_US.UTF-8 instead,  and now text is visibile. Good!. 
Now I have to investigate how solve the problem, some soft links to add? Thanks for help

**Comment by pierre-labastie** [comment]
@diamond2016: No idea... It could be an issue with Qt5 installation, but LXQt uses it, and translations are OK for LXQt.

**Comment by diamond2016** [comment]
Solved with sddm-0.16.0

**Comment by frankbe** [comment]
I had same issue today on a complete new arch linux installation with sddm 0.18 and lxqt 0.13.
It was a archlinux/archlinux vagrant box image, available on app.vagrantup.com. 

The solution was to install the package "ttf-dejavu" and reboot.

**Comment by mxgrin** [comment]
Thanks, frankbe. You made my day;)

**Comment by gtim** [comment]
Thank you @frankbe! I had the same issue with `LANG=en_US.UTF-8` on a fresh Arch install with sddm 0.19.0-6 and lxqt 0.17.0-1. Installing ttf-dejavu resolved it.

**Comment by frankbe** [comment]
Glad it helped you guys

**Comment by 0x8t** [comment]
can anyone help me too? i tried installing ttf-dejavu but it still isnt working 
![Screenshot 2024-11-17 194739](https://github.com/user-attachments/assets/99c69797-63e7-43c6-9105-ec3b80894314)

**Comment by aikidd** [comment]
Not a fix but I worked around the invisible text problem by installing sddm-theme-breeze. The breeze theme login page graphics make seeing the user name unnecessary in my case.
